### PR TITLE
Docs redirects

### PR DESCRIPTION
**Pull request by nicknovitski**
# Why

There should be an easy and obvious way to add server-side redirects and preserve extant links.

### CODE REVIEW

**Comment by brentvatne** [comment]
is there any way to add redirects for an entire subdirectory, eg: 

```
redirects[versions/latest/guides/(.*)]=versions/latest/renamed-guides/$1
```

File: docs/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
```
Comment (ide) [inline docs/deploy.sh]: Maybe add an assertion that cwd is the docs directory

**Review by ide — APPROVED** [review]

**Comment by nicknovitski** [comment]
@brentvatne You'd do it in its own loop.  Something like: `for f in versions/latest/renamed-guides/*; do redirects[versions/latest/guides/$f]=versions/latest/renamed-guides/$f; done`.

**Comment by nicknovitski** [comment]
@brentvatne You'd do it in its own loop.  Something like: `for f in versions/latest/renamed-guides/*; do redirects[versions/latest/guides/$f]=versions/latest/renamed-guides/$f; done`.

File: docs/deploy.sh (hunk 1 above)
Comment (nicknovitski) [reply docs/deploy.sh]: I used explicit paths relative to `scriptdir=$(dirname ${BASH_SOURCE[0]})`.

**Pull request merged**